src/explorer/FileTreeItem.ts

```diff
@@ -10,11 +10,18 @@ export class FileTreeItem implements IAzureTreeItem {
     public static contextValue: string = 'file';
     public readonly contextValue: string = FileTreeItem.contextValue;
     public readonly commandId: string = 'appService.showFile';
+    private _etag: string | undefined = undefined; // cannot create _etag on creation due to Kudu API calls
 
     constructor(readonly siteWrapper: SiteWrapper, readonly label: string, readonly path: string) {
     }
 
     public get id(): string {
         return `${this.siteWrapper.id}/File`;
     }
+    public get etag(): string {

```

Comment:
nit: Why not just do `public etag: string | undefined;`

Reply:
Because I have to set the etag in a different class, I thought that it made sense to have getter/setter methods.  Should I just get rid of the private property?